docs/source/alfacase_definitions/CasingSectionDescription.txt

```diff
@@ -12,10 +12,10 @@
             outer_diameter: \ :class:`Scalar <barril.units.Scalar>`\ |space|
             inner_diameter: \ :class:`Scalar <barril.units.Scalar>`\ |space|
             inner_roughness: \ :class:`Scalar <barril.units.Scalar>`\ |space|
```

Comment:
Did you try changing
```
           inner_roughness: \ :class:`Scalar <barril.units.Scalar>`\ |space|
```
to
```
           inner_roughness: \ :class:`Scalar <barril.units.Scalar>`
```
BTW, I assume `\` is the escape character.
Why are you escaping those spaces?
It that is really the escape character merging lines is what happens when you escape the newline character.
What will happen if you just remove all those escapes? If that is layout you are already using some custom css.

Reply:
As we discussed on chat, commit 9dfefcd fix this issue.